rwengine/src/render/GameRenderer.cpp

```diff
@@ -341,20 +341,18 @@ void GameRenderer::renderWorld(GameWorld* world, const ViewCamera& camera,
     culled += objectRenderer.culled;
     renderer->pushDebugGroup("Objects");
     renderer->pushDebugGroup("RenderList");
-
     // Also parallelizable
     RW_PROFILE_BEGIN("Sort");
     std::sort(renderList.begin(), renderList.end(),
               [](const Renderer::RenderInstruction& a,
                  const Renderer::RenderInstruction& b) {
-                    if (!a.drawInfo.blend && b.drawInfo.blend) return true;
-                    if (a.drawInfo.blend && !b.drawInfo.blend) return false;
+                    if (a.drawInfo.blendMode==BLEND_NONE && b.drawInfo.blendMode!=BLEND_NONE) return true;
+                    if (a.drawInfo.blendMode!=BLEND_NONE && b.drawInfo.blendMode==BLEND_NONE) return false;
```

Comment:
Interestingly, we could probably even use the actual value of the blendMode as the sort-key now (or a lookup table to re-order the blendMode values in case we re-order them).

Please do *not* change this in this PR. It's just something I wanted to note for the future.
Might be an idea to create an issue about this.